src/sample/java/samples/Browser.java

```diff
@@ -11,14 +11,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.strangeberry.jmdns.tools;
+package samples;

```

Comment:
`samples` is imho a weird package name. I had suggested `javax.jmdns.tools` on the other PR, so that we stick to the same package name as for the library code. Does anything speak against that?

Reply:
Nope I was just following the current convention of they are all"samples" so I thought the idea was to fix it in a new PR?  Right now half would be samples and have would tools?

Reply:
Ah, I see, thanks! I wasn't even aware of the current `https://github.com/jmdns/jmdns/tree/main/src/sample/java/samples` folder. 😄 
But now the PR at least leave it as it was. We can still discuss what to do with all the files in the samples folder.